drag_and_drop_v2/translations/en/LC_MESSAGES/text.po

```diff
@@ -103,163 +103,167 @@ msgstr ""
 msgid "Good work! You have completed this drag and drop problem."
 msgstr ""
 
-#: drag_and_drop_v2.py:63
+#: drag_and_drop_v2.py:75
```

Comment:
so this was the entire problem (this file), it was not properly updated in Transifex, and when I try to do that I get:
```
$ tx push -s
tx ERROR: Error received from server: Only maintainers are allowed to update the source file.
```
Could I get some help from someone at edx here? (the idea is to go to this PR and run `tx push -s`), CC @symbolist @bradenmacdonald @s0b0lev

Reply:
@kdmccormick could you please give us a hand here? :smile:

Reply:
@natabene could you please help with this? I need someone from edX (with transifex maintainer access) to run this command inside this branch

```
tx push -s
```

That is right now a blocker for this contribution as I need to change more after that's done.

Reply:
I can give you Transifex access, if you want it?

Reply:
@natabene that would be awesome, would it be better to create a generic OpenCraft account instead of a personal one for me only?

Reply:
Up to you.

Reply:
@natabene ok perfect, please add the `help@opencraft.com` account as a maintainer (if there is a difference in resources, for this case, we should be able to maintain `XBlocks.drag-and-drop-v2`

Reply:
done, please verify.

Reply:
it worked!! :tada: ty so much @natabene  :+1: